src/main.py

```diff
@@ -13,19 +10,57 @@
 If an ARTag exists, the vehicle will become a follower vehicle.
 If an ARTag does not exist, the vehicle will become the leader vehicle.
 """
+
+
+
+from follower import Follower
+from leader import Leader
+from tagrec import TagRecognition
```

Comment:
I think there's supposed to be a blank line between our imports and Python imports.